_docs-2-0/getting-started/clients.md

```diff
@@ -16,92 +16,79 @@ If you are using Maven to create Accumulo client code, add the following to your
+    props.put("instance.zookeepers", "zookeeper1,zookeeper2")
+    props.put("auth.method", "password")
+    props.put("auth.username", "myuser")
+    props.put("auth.password", "mypassword")
+    Connector conn = Connector.builder().usingProperties(props).build();
+    ```
+
+If a `accumulo-client.properties` file or a Java Properties object is used to create a [Connector], the following
+[client properties][client-props] must be set:
+
+* [instance.name]
+* [instance.zookeepers]
+* [auth.method]
+* [auth.username]
+* [auth.password]
+
+# Authentication
+
+When creating a [Connector], the user must be authenticated using one of the following
+implementations of [AuthenticationToken] below:
+
+1. [PasswordToken] is the must commonly used implementation.
+1. [CredentialProviderToken] leverages the Hadoop CredentialProviders (new in Hadoop 2.6).
+   For example, the [CredentialProviderToken] can be used in conjunction with a Java KeyStore to
+   alleviate passwords stored in cleartext. When stored in HDFS, a single KeyStore can be used across
+   an entire instance. Be aware that KeyStores stored on the local filesystem must be made available
+   to all nodes in the Accumulo cluster.
+1. [KerberosToken] can be provided to use the authentication provided by Kerberos. Using Kerberos
+   requires external setup and additional configuration, but provides a single point of authentication
+   through HDFS, YARN and ZooKeeper and allowing for password-less authentication with Accumulo.
+
+    ```java
+    KerberosToken token = new KerberosToken();
+    Connector conn = Connector.builder().forInstance("myinstance", "zookeeper1,zookeper2")
+                        .usingCredentials(token.getPrincipal(), token).build();
+    ```
 
-```java
-String instanceName = "myinstance";
-String zooServers = "zooserver-one,zooserver-two"
-Instance inst = new ZooKeeperInstance(instanceName, zooServers);
-
-Connector conn = inst.getConnector("user", new PasswordToken("passwd"));
-```
-
-The [PasswordToken] is the most common implementation of an [AuthenticationToken].
-This general interface allow authentication as an Accumulo user to come from
-a variety of sources or means. The [CredentialProviderToken] leverages the Hadoop
-CredentialProviders (new in Hadoop 2.6).
+## Writing Data
 
-For example, the [CredentialProviderToken] can be used in conjunction with a Java
-KeyStore to alleviate passwords stored in cleartext. When stored in HDFS, a single
-KeyStore can be used across an entire instance. Be aware that KeyStores stored on
-the local filesystem must be made available to all nodes in the Accumulo cluster.
+With a [Connector] created, it can be used to create objects (like the [BatchWriter]) for
+reading and writing from Accumulo:
 
 ```java
-KerberosToken token = new KerberosToken();
-Connector conn = inst.getConnector(token.getPrincipal(), token);
+BatchWriter writer = conn.createBatchWriter("table", new BatchWriterConfig())
```

Comment:
Could use the new method that does not take a BatchWriterConfig argument

Reply:
Fixed in de495ff2c